packages/server/src/core/internals/getParseFn.ts

```diff
@@ -1,3 +1,4 @@
+import { parseAsync } from 'valibot';
```

Comment:
Will this treeshake away or will it cause runtime errors cause it can't be found for projects not using Valibot?

Reply:
I do not know. That's why I mentioned it in the description.

Reply:
I can imagine that it could cause problems. An alternative would be a dynamic import. However, I have no experience with it.

Reply:
It might mean that the API must change for valibot users to

```ts
import * as v from "valibot";

const myProc = publicProceudre
  .input(v.parseAsync(
    v.object({ ... })
  .mutation(...)
```

Reply:
I think this doesn't work because the value being validated is the 2 parameter of the parse function.

Reply:
Another alternative would be to offer an adapter that can import Valibot.

Reply:
Sorry, I didn't look closely at your code example yesterday. My previous statements are wrong. I think your idea is good. I'll check it in the next days and get back to you.